netbench/netbench-cli/etc/Dockerfile

```diff
@@ -64,7 +68,14 @@ RUN set -eux; \
     cp target/debug/netbench-cli /app/bin; \
   fi; \
   rm -rf target; \
-  echo "#!/usr/bin/env bash\naws s3 cp s3://\$S3_BUCKET/client.json ./client.json\naws s3 cp s3://\$S3_BUCKET/server.json ./server.json\neval /usr/bin/netbench-cli ./client.json ./server.json \$@ > report.json\naws s3 cp ./report.json s3://\$S3_BUCKET/" > /app/bin/start; 
+  echo "#!/usr/bin/env bash\n \
+  aws s3 cp s3://\$S3_BUCKET/client.json /tmp/client.json\n \
+  aws s3 cp s3://\$S3_BUCKET/exportedlogs/\$EXPORT_TASK_ID/ /tmp/ --recursive\n \
+  export DIR=\$(ls -d /tmp/*/)\n \
+  printenv\n \
+  gzip -d \$(echo \$DIR)000000.gz && sort -k1 \"\$(echo \$DIR)000000\" > \$(echo \$DIR)000000_sorted && python3 /usr/bin/logParser.py \$(echo \$DIR)000000_sorted && mv \$(echo \$DIR)000000_sorted /tmp/server.json\n \
```

Comment:
Probably worth splitting this up into multiple lines. You can fail the whole script by setting `set -e` at the top